torch/utils/data/dataloader.py

```diff
@@ -272,38 +254,44 @@ class DataLoader(object):
             at every epoch (default: False).
         sampler (Sampler, optional): defines the strategy to draw samples from
             the dataset. If specified, the ``shuffle`` argument is ignored.
+        batch_sampler (Sampler, optional): like sampler, but returns a batch of
```

Comment:
It would be nice to raise an error when both batch_sampler and one of the ignored options is specified. Just change their default values to None and add checks